### DISCUSSION TITLE
Relation "serverpod_user_info" does not exist

### DISCUSSION THREAD

**Discussion by MathiasCochet** [serverpod/serverpod#814] [Q&A] [answered]
**Describe the bug**
When trying to authenticate with google sign, I get an error for a non-existing database reference

**To Reproduce**
Follow the guide for setting up google sign in on Android

**Serverpod versions**
1.0.1

**Additional context**
`METHOD CALL: serverpod_auth.google.authenticateWithServerAuthCode duration: 343ms numQueries: 0 authenticatedUser: null
PostgreSQLSeverity.error 42P01: relation "serverpod_user_info" does not exist
package:postgres/src/connection.dart 514:18                              _PostgreSQLExecutionContextMixin._query
package:postgres/src/connection.dart 475:7                               _PostgreSQLExecutionContextMixin.query
package:postgres/src/connection.dart 543:22                              _PostgreSQLExecutionContextMixin.mappedResultsQuery
package:postgres_pool/postgres_pool.dart 820:23                          _PgExecutionContextWrapper.mappedResultsQuery.<fn>
package:postgres_pool/postgres_pool.dart 746:27                          _PgExecutionContextWrapper._run
package:postgres_pool/postgres_pool.dart 819:12                          _PgExecutionContextWrapper.mappedResultsQuery
package:postgres_pool/postgres_pool.dart 661:16                          PgPool.mappedResultsQuery.<fn>
package:postgres_pool/postgres_pool.dart 352:22                          PgPool.run.<fn>.<fn>
package:postgres_pool/postgres_pool.dart 451:27                          PgPool._useOrCreate
package:postgres_pool/postgres_pool.dart 423:14                          PgPool._withConnection.<fn>
package:executor/src/executor_impl.dart 61:19                            _Executor.scheduleTask
package:postgres_pool/postgres_pool.dart 351:18                          PgPool.run.<fn>
package:retry/retry.dart 131:16                                          RetryOptions.retry
package:postgres_pool/postgres_pool.dart 349:14                          PgPool.run
package:serverpod/src/database/database_connection.dart 171:20           DatabaseConnection.find
package:serverpod/src/database/database_connection.dart 199:18           DatabaseConnection.findSingleRow
package:serverpod/src/database/database.dart 86:12                       Database.findSingleRow
package:serverpod_auth_server/src/business/users.dart 36:12              Users.findUserByEmail
package:serverpod_auth_server/src/endpoints/google_endpoint.dart 191:20  GoogleEndpoint._setupUserInfo
package:serverpod_auth_server/src/endpoints/google_endpoint.dart 66:20   GoogleEndpoint.authenticateWithServerAuthCode
package:serverpod/src/server/endpoint_dispatch.dart 111:20               EndpointDispatch.handleUriCall
package:serverpod/src/server/server.dart 224:18                          Server._handleRequest
===== asynchronous gap ===========================
package:postgres_pool/postgres_pool.dart 351:18                          PgPool.run.<fn>
package:retry/retry.dart 131:16                                          RetryOptions.retry
package:postgres_pool/postgres_pool.dart 349:14                          PgPool.run
package:serverpod/src/database/database_connection.dart 171:20           DatabaseConnection.find
package:serverpod/src/database/database_connection.dart 199:18           DatabaseConnection.findSingleRow
package:serverpod/src/database/database.dart 86:12                       Database.findSingleRow
package:serverpod_auth_server/src/business/users.dart 36:12              Users.findUserByEmail
package:serverpod_auth_server/src/endpoints/google_endpoint.dart 191:20  GoogleEndpoint._setupUserInfo
package:serverpod_auth_server/src/endpoints/google_endpoint.dart 66:20   GoogleEndpoint.authenticateWithServerAuthCode
package:serverpod/src/server/endpoint_dispatch.dart 111:20               EndpointDispatch.handleUriCall
package:serverpod/src/server/server.dart 224:18                          Server._handleRequest`

**Comment by yahu1031** [comment]
@MathiasCochet did you generate/added the table in DB?

**Reply by MathiasCochet** [reply]
You mean the initial setup, or do I need to do something after setting up authentication ?

**Reply by Isakdl** [reply, marked as answer]
The documentation could be more clear but you have to add the tables from the serverpod_auth module to your database!

You can find the pgsql file with all the setup required here: https://github.com/serverpod/serverpod/blob/main/modules/serverpod_auth/serverpod_auth_server/generated/tables.pgsql

Run this SQL code in your database and you should be good to go.
